documentation/public/user-guide/08-examples/index.md

```diff
@@ -0,0 +1,21 @@
+# 8. Examples
+
+- [8.1. List functions in a contract](./01-list-functions-in-contract/index.md)
+- [8.2. Find usages](./02-find-usages/index.md)
+- [8.3. Jump to definition](./03-jump-to-definition/index.md)
+
+Throughout the examples in this section we will use a sample contract. We need some setup code to [create a `CompilationUnit`](../07-semantic-analysis/01-compilation-units/index.md) with the contents of the contract:
```

Comment:
TOL: I wonder if we should embed these in the relevant sections instead, to make sure that each example is self contained?

Reply:
Not sure about this. On the one hand, it'd be nice to have everything self contained, but on the other, it's a lot of duplicated code.